src/components/Tag/core/tag.helpers.tsx

```diff
@@ -0,0 +1,21 @@
+import s from './tag.module.scss'
+import { IProps } from './tag.typings'
+
+export const getClassNames = ({
+  color,
+  className,
+}: Pick<IProps, 'className'> & Pick<Required<IProps>, 'color'>) => {
+  const classNames: string[] = [
+    s['tag'],
+    s[`tag-${color}`],
+    'f',
```

Comment:
I think you might have forgotten to horizontally center the children of the flexbox :

```suggestion
    'f',
    'f-jc-center'
```

Reply:
Thank you!